.github/workflows/get_layer_info_for_nf_imgs.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          ref: ${{ github.head_ref }}
+          ref: store/branch_to_track_nf_img_layers
```

Comment:
wait won't this use the `allowed_base_images.txt` file from the branch which could be outdated?

Reply:
nvm we get it from master branch